testing/trino-product-tests/src/main/java/io/trino/tests/product/iceberg/TestIcebergSparkCompatibility.java

```diff
@@ -423,6 +424,33 @@ public void testTrinoShowingSparkCreatedTables()
         onTrino().executeQuery("DROP TABLE " + trinoTableName(trinoTable));
     }
 
+    @Test(groups = {ICEBERG, PROFILE_SPECIFIC_TESTS})
+    public void testPrestoWritingDataWithObjectStorageLocationProvider()
+    {
+        String baseTableName = "test_object_storage_location_provider";
+        String sparkTableName = sparkTableName(baseTableName);
+        String trinoTableName = trinoTableName(baseTableName);
+        String dataPath = "hdfs://hadoop-master:9000/user/hive/warehouse/test_object_storage_location_provider/obj-data";
+
+        String sparkTableDefinition = "CREATE TABLE %s (_string STRING, _bigint BIGINT) USING ICEBERG OPTIONS (" +
```

Comment:
is using `OPTIONS` different from using `TBLPROPERTIES`?

Reply:
it's the same. I can change to TBLPROPERTIES for consistency.